nntrainer/layers/loss/loss_layer.cpp

```diff
@@ -25,10 +25,6 @@ void LossLayer::finalize(InitLayerContext &context) {
                     nntrainer::TensorDataTypeInfo>::from_string("FP32"));
 
   context.setOutputDimensions(output_dim);
-
-  is_inplace = true;
-  if (context.getActivationDataType() != ml::train::TensorDim::DataType::FP32)
-    is_inplace = false;
```

Comment:
I understood that this code is meaningless (based on your description). Then, which part do this action? I cannot find the part replacing this code to support mixed precision. Could you explain this?

Reply:
@EunjuYang Currently, processing for the input layer is being done within the network graph, but processing for the loss layer has not been implemented yet. The loss layer cannot be processed in the same way as the input layer. I'll try to set whether or not the loss layer is in-place and upload it as a separate PR. Very good point. Thanks a lot!

Reply:
Oh I see! Thank you the detailed explanation !👍